### ISSUE TITLE
Discussion of readings for session 3: 3D imaging in museums

### ISSUE THREAD

**Issue by gabrielbodard** [SunoikisisDC/SunoikisisDC-2021-2022#19]
1. Davide Tanasi, Ilenia Gradante & Stephan Hassam. 2019. "Best Practices for 3D Digital Recording and Global Sharing of Catacombs from Late Roman Sicily." _Studies in Digital Heritage_ 3.1. Pp. 60.82. Available: http://dx.doi.org/10.14434/sdh.v3i1.25290
2. Anca Timofan et al (2019). "PANTHEON 3D: An Initiative in the Three-Dimensional Digitization of Romanian Cultural Heritage." _Studia Universitatis Babes-Bolyai Digitalia_ 63.2, pp. 65-83. Available: https://digihubb.centre.ubbcluj.ro/journal/index.php/digitalia/article/view/52

**Comment by lettychardon** [comment]
The Timofan et al. article raises a number of very interesting points beyond merely describing the details of the project and its purpose. The first is the emphasis on the creation and display of 3D models to enhance the museum-going experience of the vision impaired. This has been a long time coming, but for obvious reasons ancient artefacts cannot be directly handled by visitors, so to provide a replica which can be held is fantastic! The authors also describe this development as "restoring some essential aspects into the cultural experiences provided, like tactile and multi-sensorial interaction." This is not something that I have really thought of before - these objects would all have been within reach in their original contexts but have now lost that element. The authors also mention that the digitisation of the contents of the museum could lead to the promotion of Romania. Romania probably isn't the biggest tourist destination for those interested in the Roman era but there is a possibility that people could discover the collections online and then to decide to visit Romania for real and discover something they had never even thought of before. Lastly, I very much enjoyed the fact that they took both the researcher and a more general audience into consideration, encompassing the past, the present and the future.

**Comment by molmay** [comment]
> The first is the emphasis on the creation and display of 3D models to enhance the museum-going experience of the vision impaired. This has been a long time coming, but for obvious reasons ancient artefacts cannot be directly handled by visitors, so to provide a replica which can be held is fantastic! 

I also found the use of 3D printing to make exhibits more accessible to the visually impaired really fascinating as well - it is a shame that this technology is not currently available in a more widespread fashion because it would greatly enhance the museum experience for so many! 

The Timofan article also really piqued my interest in its description of 3D models being used in ‘worldwide educational experiences’ and therefore the implication that this technology could help end some of the bias in favour of certain locations for academia or access to education by allowing for the exchange of physical objects between locations. I love the use of the word ‘experience’ in this context as well, it really highlights how this technology transforms ancient material into something that can really be engaged with rather than learnt about, just like Letty has commented about interaction!

**Comment by gabrielbodard** [comment]
It's worth highlighting, as I think I mentioned in the video last week, that there has been some hostility to the use of 3D printing for accessibility for blind and partially sighted visitors among disability circles. I think there experience has been that (some) museums have decided there's no need to make any other accommodations and just fobbed blind users off with a 3D print of the object (which doesn't represent the texture, weight, feel, accurate form etc. of the object very well). So while I do think it's a useful tool, I don't think it can been taken as *enough*… and shouldn't discourage institutions from going further.

**Comment by molmay** [comment]
I totally agree! Hopefully it's a technology that museums will use in conjunction with other accessibility measures to replicate the experience for those with visually impairments.

**Comment by JasNewtonRae** [comment]
I found it really interesting to read about the different kinds of 3D imaging within the Tanasi article and how different reasons for using them and their pros and cons. I also really appreciate the fact that 3D imaging allows access to sites even when they are closed between excavations.

In terms of the Timofan article I agree that the 3D printing of the sculptures is a huge step forwards in educational materials and accessibility but there still definitely needs to be more done. It should not just be left as is  but expanded on.

**Comment by Avieava** [comment]
In terms of accessibility this is not a perspective I have ever considered. Firstly the ability for artefact replicas to transgress global boundaries is an important development in the academic world as information can be shared all over. Furthermore the ability to share art and information with the visually impaired is a great but development as Jasmine mentioned it would be interesting to see if any further progression has been made or considered as this was not mentioned.

**Comment by Avieava** [comment]
I wonder if from an environmental point of view 3D scanning could be used for more precise digs that interrupt the environment around less? Or similarly from an ethical point of view (I’m writing my dissertation on the exhibition of human remains) could this be a way to respectfully study burial sights without interrupting them in any way?

**Comment by Ghilaevansky** [comment]
Regarding the Tanasi article, I really enjoyed reading about the catacombs and found it extremely interesting. It is amazing how these technologies exist and the power of 3D scanning and its affect on remains and presenting history in a new way. It creates in depth accessibility to sites, which is incredible. I didn't realise the limitations that exist in 3D scanning, as mentioned in the article: damaged equipment, poor lighting and the presence of water. 

The Timofan article made me want to observe an excavation and 3D scanning of a site! I can't help but wonder if there is a danger in 3D scanning, for example, someone could make false replicas of replicas and then try to sell them or pass them off as the original object? On the other hand, it gives children a chance to play with the objects and get that personal experience, without touching the real artefact and risking damage to it.

**Comment by portableant** [comment]
Worth looking at the work of Rafie Cecilia https://discovery.ucl.ac.uk/id/eprint/10140706/ re the accessibility issue. We've used 3d prints for handling and teaching since 2014, and mostly with Steve Dey of ThinkSee3d for realistic prints and casts. Maybe also have a look through the papers here as well https://creative-economy.fitzmuseum.cam.ac.uk/conference/papers/

**Comment by ellie919** [comment]
Something which struck me was the ability for digital scanning to reveal features which are not available to the naked eye. For example, the way in which laser scanning revealed many layers of paint on the Fresco of the Little Commissioner in the Catacombs of St Lucy. This was touched on in the seminar, with the layers of the ancient Egyptian funerary mask and the obscured cuneiform tablets. In addition to providing access to ancient sites which cannot be opened to the public (e.g. due to safety concerns over flooding or the fragility of the remains), certain methods of 3D imaging can reveal how artefacts were made, altered, or obscured in time without compromising the preservation of the object. In Hittite, we usually look at drawings of tablets which were done by hand and are often inaccurate. A 3D image of the texts would help to eradicate these human errors which can change the meaning of the writing, as well as preserving the inherently 3-dimensional nature of the text.

**Comment by gabrielbodard** [comment]
3D imaging of Cuneiform tablets is a lovely example of the value of 3D technologies, yes. They're often imaged using RTI, but a full 3D model (and therefore printable object) is also really valuable.
